src/server.tsx

```diff
@@ -41,27 +42,37 @@ app.get('/healthcheck', (req: express.Request, res: express.Response) => {
     res.send('OK');
 });
 
-interface Response {
-    html: string;
-    css: string;
-    js: string;
-    meta: EpicTestTracking;
+interface DataResponse {
+    data?: {
+        moduleUrl: string;
+        variant: Variant;
+        meta: EpicTestTracking;
+    };
+    debug?: Debug;
+}
+
+interface MarkupResponse {
+    data?: { html: string; css: string; js: string; meta: EpicTestTracking };
     debug?: Debug;
 }
 
 const [, fetchConfiguredEpicTestsCached] = cacheAsync(fetchConfiguredEpicTests, 60);
 
-const asResponse = (component: JsComponent, meta: EpicTestTracking, debug?: Debug): Response => {
+const asResponse = (
+    component: JsComponent,
+    meta: EpicTestTracking,
+    debug?: Debug,
+): MarkupResponse => {
     const { el, js } = component;
     const { html, css } = extractCritical(renderToStaticMarkup(el));
-    return { html, css, js, meta, debug };
+    return { data: { html, css, js, meta }, debug };
 };
 
 const buildEpic = async (
```

Comment:
Suggestion: as we're adopting the data vs markup terminology, maybe rename this to `buildEpicMarkup`?